### PR TITLE
G0 6911

### PR DESCRIPTION
**Pull request by sadik-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/userDetails/userDetails.module.ts [hunk 1]
```diff
@@ -60,6 +61,7 @@ const DEFAULT_PERFECT_SCROLLBAR_CONFIG: PerfectScrollbarConfigInterface = {
         DecimalDigitsModule,
         SharedModule,
         ElementViewChildModule,
+        AccordionModule.forRoot(),
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/userDetails/userDetails.module.ts:64]: If we are not using accordian, remove this.

File: apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.ts [hunk 2]
```diff
@@ -47,6 +47,7 @@ export class SubscriptionsComponent implements OnInit, AfterViewInit, OnDestroy
     public currentCompanyPlan: any = '';
     public activityLogAsideMenuState: string = 'out';
     public companyDetailsAsideMenuState: string = 'out';
+    oneAtATime: boolean = true;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.ts:50]: Consider using access modifier **public**

File: apps/web-giddh/src/app/userDetails/userDetails.module.ts (hunk 1 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/app/userDetails/userDetails.module.ts]: fixed

File: apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.ts (hunk 2 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.ts]: code removed

**Pull request merged**